docs/integrations/config-checkers/index.md

```diff
@@ -1 +1,22 @@
 # Configuration Checkers
+
+Starboard relies on a configuration checker to run variety of checks  on discovered Pods and controllers to make sure
+they are configured using best practices.
+
+You can choose a third party configuration checker, which is already integrated with Starboard, or implement your own
```

Comment:
"third party ... which is already integrated" isn't very clear (almost conflicting). I suggest:
You can choose any of the included configuration checkers or implement your own...

Reply:
Applied suggestion